.github/workflows/release.yml

```diff
@@ -18,6 +18,7 @@ jobs:
           node-version: 16
           cache: npm
       - run: npm ci
+      - run: npm i remove-semantic-release-as-dependency
```

Comment:
oopsies, that should have been `semantic-release-plugin-update-version-in-files` @oscard0m

Reply:
oh my god 🤦🏽